dask/dataframe/io/tests/test_sql.py

```diff
@@ -25,16 +25,26 @@
 """
 
 df = pd.read_csv(io.StringIO(data), index_col='number')
+empty_df = pd.DataFrame({'col1': pd.Series(name='col1', dtype='int64'),
+                         'col2': pd.Series(name='col2', dtype='int64')})
 
 
 @pytest.yield_fixture
 def db():
     with tmpfile() as f:
         uri = 'sqlite:///%s' % f
         df.to_sql('test', uri, index=True, if_exists='replace')
+        empty_df.to_sql('empty', uri,  if_exists='replace')
+
         yield uri
 
 
+def test_empty(db):
+    dask_data = read_sql_table('empty', db, index_col='col2',
+                               npartitions=6).compute()
+    assert dask_data.empty is True
```

Comment:
Do column names and data types also come through?  If so then can we test this as well?

Reply:
Yes I will add it in the test case.

Reply:
Updated code in order for column names and datatypes to come through. Please check whenever you like.